src/NUnitFramework/framework/Internal/TestExecutionContext.cs

```diff
@@ -484,9 +484,19 @@ private TestExecutionContext CreateIsolatedContext()
             return context;
         }
 
-#endregion
+        /// <summary>
+        /// Send a test message to listeners
+        /// </summary>
+        /// <param name="destination">Destination of test message to be handled by listeners</param>
```

Comment:
How about `A name recognized by the intended listeners.` or something like that?